src/App.vue

```diff
@@ -1,22 +1,73 @@
 <template>
-  <nav class="navbar navbar-expand-sm navbar-dark bg-dark">
-    <div class="container-fluid">
 
-   
-    <router-link to="/" class="navbar-brand">
+<!-- <nav class="navbar navbar-expand-sm bg-dark navbar-dark fixed-top">
```

Comment:
Remove unused/commented code and include comments about functionality in simple words